crates/starknet-devnet-core/src/constants.rs

```diff
@@ -34,10 +34,10 @@ pub const ETH_ERC20_CONTRACT_CLASS: &str = include_str!(concat!(
     "/contracts/system_artifacts/erc20_eth.sierra"
 ));
 
-pub const STRK_ERC20_CONTRACT_ADDRESS: Felt =
```

Comment:
To minimize the footprint of this PR, this can be reverted

Reply:
I had same thought, but in the end I left it because it ensures consistent ordering of consts for ETH and STRK contracts. HASH > ADDRESS > CLASS. Even though I see now that HASH > CLASS > ADDRESS might be better in both cases.